src/Packages/Packages.csproj

```diff
@@ -62,7 +62,12 @@
       <Link>WebJobs.Extensions.ApiHub\WebJobs.Extensions.ApiHub.nuspec</Link>
     </NuGetSpec>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Reference Include="Microsoft.WindowsAzure.Storage">
```

Comment:
This is just VS's nugget upgrade. These shouldn't be changing. I'll investigate.